### ISSUE TITLE
2021-04-29 Performance regression for Norne with MSW

### ISSUE THREAD

**Issue by blattms** [OPM/opm-simulators#3227]
According to [Phoronics benchmarking](https://linuxbenchmarking.com/?open-porous-media-git) we introduced a regressen around April 29  in the model it calls Norne-4C MSW. Any ideas where that comes from?

**Comment by alfbr** [comment]
Yes, I believe it came here:
https://github.com/OPM/opm-simulators/pull/3183
and it was due to instabilities. That PR was thoroughly tested before merged, and the differences in runtime was with high probability random.  I expect the increase in runtime to be nullified by the other PRs with multiple still in flight.

**Comment by totto82** [comment]
Yes. The Norne 4C test is very unstable at the moment due to some problematic wells. I have a long series of PRs that is supposed to make it more stable and faster. I believe the actual regression is caused by https://github.com/OPM/opm-simulators/pull/3183. It will be some bumping up and down for the Norce 4C case for some time now.

**Comment by blattms** [comment]
This is ancient. Closing.
